common/cf-solana/src/client/impls.rs

```diff
@@ -486,8 +486,8 @@ impl ClientState {
         host_timestamp: ibc::Timestamp,
     ) -> bool {
         let expiry_ns = consensus
-            .timestamp_sec
-            .get()
+            .timestamp()
+            .nanoseconds()
```

Comment:
So was this a bug before? Since timestamp_sec implies it was in seconds but we didn’t multiply it to get nanoseconds? If this is a bug fix, could you mention that and details of it in the PR description?

Reply:
its a bug fix. ill mention in the PR